src/cactus.js

```diff
@@ -134,28 +74,26 @@ async function cutReleaseBranch(args) {
 
   // Shell out to run npm version inside tempdir
   cp.execSync(`npm version ${args.level} -m "Release v%s"`, { cwd: tmpdir.name });
-  repo.refreshIndex();
 
   if (!approval.lgtm) {
     return 'Aborted branch cut! Phew, that was a close one...';
   }
 
   // Push master, the release branch, and tag
   logger.info(`Pushing branch ${versionInfo.releaseBranchName} & tag v${versionInfo.version}`);
-  await clonedRemote.push([
-    `refs/heads/master:refs/heads/master`,
-    `refs/heads/master:refs/heads/${versionInfo.releaseBranchName}`,
-    `refs/tags/v${versionInfo.version}:refs/tags/v${versionInfo.version}`,
-  ], makeGitOpts(credentials));
+
+  await clonedRepo.push('origin', 'master');
```

Comment:
add a comment explaining this (for an arbitrary repo users could change this alias name, but since it's a fresh clone it's always named origin)

Reply:
(the alternative would be to see if there's an alternative more akin to the old code, but I don't see a failure case for the origin way of doing things)